metadata.rb

```diff
@@ -10,6 +10,12 @@
   supports os
 end
 
-%w{ yum apt python logrotate }.each do |ckbk|
+case node["platform"
+when "fedora", "centos", "redhat"
+  depends yum
+end
+
+
+%w{  apt python logrotate }.each do |ckbk|
```

Comment:
Now that we're at it.. maybe we shouldn't let it depend on `apt` if we're on `"fedora", "centos", "redhat"`?